pages/feedback.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'; // React
+import Layout from '@components/Layout'; // Layout
+import Title from '@components/Title'; // Title
+import Button from '@components/Button'; // Button
+import Table from '@components/Table'; // Table
+import Checkbox from '@components/Checkbox'; // Checkbox
+import Modal from '@components/Modal'; // Modal
+import Event from '@containers/Event'; // Event state
+import styles from '@styles/pages/Feedback.module.scss'; // Component styles
+
+const SAMPLE_DATA = [
+  {
+    schoolName: 'CCI',
```

Comment:
Why not just pull this directly from the schools api?

Reply:
Going to leave it for the API hookup.